package com.protos;

option java_package = "com.protos";
option java_outer_classname = "CardProto";

message Card {
  required bool has_been_used = 1;
  required bool in_hand = 2;
  required string name = 3;
}